programs/stake/src/stake_state.rs

```diff
@@ -2288,7 +2288,7 @@ mod tests {
                 &Authorized::default(),
                 &Lockup::default(),
                 &Rent {
-                    lamports_per_byte_year: 42,
+                    lamports_per_byte_year: stake_lamports,
```

Comment:
nit: what do you think about having these use `Rent::default()` and instead of `42` using the standard rent-exempt amount of `2282880` lamports?  Wherever it makes sense, of course

Reply:
I agree. Done!